pkgs/tools/security/gopass/hibp.nix

```diff
@@ -32,7 +41,18 @@ buildGoModule (finalAttrs: {
 
   postFixup = ''
     wrapProgram $out/bin/gopass-hibp \
-      --prefix PATH : "${lib.makeBinPath [ gopass ]}"
+      --prefix PATH : "${gopass.wrapperPath}"
```

Comment:
And `gopass` itself is not needed?

Reply:
I noticed that `gopass-jsonapi` doesn’t wrap the gopass binary, so I followed the same approach. After removing it, the version check in gopass-hibp still works (it initializes the gopass API instance). However, keeping it there might not be a bad idea either.

Reply:
`gopass-api` imports `github.com/gopasspw/gopass/pkg/gopass/api` directly, which in turn uses `github.com/gopasspw/gopass/internal/*`. So I think it is used as a Go library and will likely be statically linked.